inference-engine/tests/functional/plugin/shared/src/single_layer_tests/gather_nd.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <memory>
+#include <random>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "common_test_utils/common_utils.hpp"
+#include "functional_test_utils/precision_utils.hpp"
+#include "functional_test_utils/skip_tests_config.hpp"
+
+#include "single_layer_tests/gather_nd.hpp"
+
+namespace IE = InferenceEngine;
+
+namespace LayerTestsDefinitions {
+
+std::string GatherNDLayerTest::getTestCaseName(const testing::TestParamInfo<gatherNDParams> &obj) {
+    gatherNDLayerParams params;
+    IE::Precision netPrc, indicesPrc;
+    IE::Layout inLayout, outLayout;
+    LayerTestsUtils::TargetDevice targetDevice;
+    std::map<std::string, std::string> config;
+
+    std::tie(params, netPrc, indicesPrc, inLayout, outLayout, targetDevice, config) = obj.param;
+
+    std::size_t batchDims;
+    IE::SizeVector inputShape, indicesShape;
+    std::tie(batchDims, inputShape, indicesShape) = params;
+
+    std::ostringstream result;
+    result << "IS=" << CommonTestUtils::vec2str(inputShape) << "_";
+    result << "indicesShape=" << CommonTestUtils::vec2str(indicesShape) << "_";
+    result << "batchDims=" << batchDims << "_";
+    result << "inPRC=" << netPrc.name() << "_";
+    result << "indicesPRC=" << indicesPrc.name() << "_";
+    result << "outPRC=" << netPrc.name() << "_";
+    result << "inL=" << inLayout << "_";
+    result << "outL=" << outLayout << "_";
+    result << "trgDev=" << targetDevice << "_";
+    return result.str();
```

Comment:
It looks like you need to add "config" to extend the test name to avoid duplication